addons/mail/static/src/core/common/message_model.js

```diff
@@ -182,6 +181,7 @@ export class Message extends Record {
     create_date = Record.attr(undefined, { type: "datetime" });
     /** @type {luxon.DateTime} */
     write_date = Record.attr(undefined, { type: "datetime" });
+    needaction = undefined;
```

Comment:
undefined is the default, it doesn't need to be specified
`@type` should be added to help